crates/ide_assists/src/handlers/replace_for_loop_with_for_each.rs

```diff
@@ -50,14 +50,15 @@ pub(crate) fn replace_for_loop_with_for_each(acc: &mut Assists, ctx: &AssistCont
                 // We have either "for x in &col" and col implements a method called iter
                 //             or "for x in &mut col" and col implements a method called iter_mut
                 format_to!(buf, "{}.{}()", expr_behind_ref, method);
+            } else if matches!(iterable, ast::Expr::RangeExpr(..)) {
```

Comment:
```suggestion
            } else if let ast::Expr::RangeExpr(..) = iterable {
```

to be consistent with a similar check below